app/models/optimized_image.rb

```diff
@@ -68,7 +68,7 @@ def self.create_for(upload, width, height, opts = {})
         Rails.logger.error("Could not find file in the store located at url: #{upload.url}")
       else
         # create a temp file with the same extension as the original
-        extension = ".#{upload.extension}"
+        extension = ".#{opts[:format] || upload.extension}"
```

Comment:
Is this option still required because I don't see it being used in this PR. Also, I think we shouldn't allow this option because the upload extension has to be based on that of the upload which is determined from the actual image that we're holding on this.

Reply:
That was a mistake of mine during a refactor (the code said png: 'png' instead of format: 'png')

For this particular feature we did a series of benchmarks and found that PNG gave the best results, so this is required to force the PNG format.